misk/web/tabs/web-actions/src/containers/WebActionsContainer.tsx

```diff
@@ -151,91 +196,208 @@ const MethodTag = (props: { method: HTTPMethod }) => (
   </FloatRight>
 )
 
+/**
+ * Collapse wrapped Send a Request form for each Web Action card
+ */
+const SendRequestCollapse = (
+  props: { action: IWebActionInternal } & IState & IDispatchProps
+) => (
+  <Collapse
+    isOpen={simpleSelect(
+      props.simpleForm,
+      `${tag}::${props.action.pathPattern}::Request`,
+      "data"
+    )}
+  >
+    <InputGroup
+      onChange={onChangeFnCall(
+        props.simpleFormInput,
+        `${tag}::${props.action.pathPattern}::URL`
+      )}
+      value={
+        simpleSelect(
```

Comment:
pretty long conditional here, is it easy to split these out into their own vars?

Reply:
Fixed in #827